examples/make_simple/code/Makefile

```
@@ -19,3 +19,4 @@ install: $(BUILD_DIR)/lib/liba.a
 	mkdir -p $(PREFIX)/lib $(PREFIX)/include
 	cp -rpv $(BUILD_DIR)/lib $(PREFIX)
 	cp -p liba.h $(PREFIX)/include
+	cp $(README_DIR)/README.md $(PREFIX)/share
```

I think line 22 is the cause of the RBE failure - it is complaining that the wrong file type is found i.e. a file is found in place of a directory as this is creating a file called share that is a copy of README.md
```suggestion
	mkdir -p $(PREFIX)/lib $(PREFIX)/include $(PREFIX)/share
	cp -rpv $(BUILD_DIR)/lib $(PREFIX)
	cp -p liba.h $(PREFIX)/include
	cp $(README_DIR)/README.md $(PREFIX)/share/
```